sdk/monitor/monitor-opentelemetry/src/logs/handler.ts

```diff
@@ -2,37 +2,41 @@
 // Licensed under the MIT license.
 
 import { AzureMonitorLogExporter } from "@azure/monitor-opentelemetry-exporter";
-import { OTLPLogExporter } from "@opentelemetry/exporter-logs-otlp-http";
 import { logs } from "@opentelemetry/api-logs";
-import {
-  LoggerProvider,
-  BatchLogRecordProcessor,
-  Logger as OtelLogger,
-} from "@opentelemetry/sdk-logs";
+import { LoggerProvider, BatchLogRecordProcessor } from "@opentelemetry/sdk-logs";
 import { LoggerProviderConfig } from "@opentelemetry/sdk-logs/build/src/types";
-import { AzureMonitorOpenTelemetryConfig } from "../shared/config";
+import { InternalConfig } from "../shared/config";
 import { MetricHandler } from "../metrics/handler";
 import { AzureLogRecordProcessor } from "./logRecordProcessor";
 
 /**
  * Azure Monitor OpenTelemetry Log Handler
  */
 export class LogHandler {
+  private static _instance: LogHandler;
   private _loggerProvider: LoggerProvider;
-  private _logger: OtelLogger;
   private _azureExporter: AzureMonitorLogExporter;
-  private _otlpExporter?: OTLPLogExporter;
   private _logRecordProcessor: BatchLogRecordProcessor;
-  private _config: AzureMonitorOpenTelemetryConfig;
+  private _config: InternalConfig;
   private _metricHandler?: MetricHandler;
   private _azureLogProccessor: AzureLogRecordProcessor;
 
+  public static getInstance(config: InternalConfig, metricHandler: MetricHandler) {
+    if (!LogHandler._instance) {
+      LogHandler._instance = new LogHandler(config, metricHandler);
+    }
+    return LogHandler._instance;
+  }
+
   /**
    * Initializes a new instance of the TraceHandler class.
    * @param _config - Distro configuration.
    * @param _metricHandler - MetricHandler.
    */
-  constructor(config: AzureMonitorOpenTelemetryConfig, metricHandler: MetricHandler) {
+  constructor(config: InternalConfig, metricHandler: MetricHandler) {
+    // if (LogHandler._instance) {
```

Comment:
Looks like this should be uncommented if you're looking to ensure you don't double-initialize your singleton.